source/operations/troubleshooting.rst

```diff
@@ -156,6 +158,41 @@ You can scroll vertically through the output for the selected check.
    :alt: MinIO SUBNET's health report for a deployment showing a failed Health Report with details expanded
    :align: center
 
+Logs
+----
+
+Use :mc:`mc admin logs` command to display logs from the command line.
+The command supports type and quantity filters for further limiting logs output.
+
+Optionally, use :ref:`Call Home <minio-troubleshooting-call-home>` to start automatically uploading real time error logs to SUBNET for analysis.
+
+.. _minio-troubleshooting-call-home:
+
+Call Home
+---------
+
+Call Home functionality automates the collection and uploading of diagnostic data or error logs to SUBNET.
+You much have an active SUBNET registration for the deployment to use Call Home.
```

Comment:
```suggestion
You must have an active SUBNET registration for the deployment to use Call Home.
```

Reply:
Perhaps we need to make it clear that this is exclusively accessible for online clusters and unavailable in airgapped mode?

Reply:
Fixed the typo and added info on airgap limitation.